frontend/src/components/TransactionEventList/Events/TokenEvents.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+	<span v-if="event.event.__typename === 'TokenTransferEvent'">
+		<span>
+			Transferred {{ event.event.amount.value }} <b>{{ event.tokenId }}</b>
+			From
+
+			<AccountLink :address="event.event.from.address" />
+		</span>
+		<span>
+			To
+			<AccountLink :address="event.event.to.address" />
+		</span>
+	</span>
+
+	<span v-else-if="event.event.__typename === 'MintEvent'">
+		<span>
+			Minted {{ event.event.amount.value }} <b>{{ event.tokenId }}</b>
+
+			To
+			<AccountLink :address="event.event.target.address" />
+		</span>
+	</span>
+	<span v-else-if="event.event.__typename === 'BurnEvent'">
+		<span>
+			Burned {{ event.event.amount.value }} <b>{{ event.tokenId }}</b>
+
+			To
```

Comment:
```suggestion
			From
```

Reply:
Hi I have updated it , Please review and let me know